lib/util/include/hse_util/logging.h

```diff
@@ -120,15 +127,24 @@
 #define log_errx(_fmt, _err, ...)   log_prix(HSE_LOGPRI_ERR, (_fmt), false, (_err), ##__VA_ARGS__)
 
 
+/* Helper APIs used by above log macros.  Not intended to be called directly.
+ */
 void
 hse_log(struct event_counter *ev, const char *fmt, bool async, void **args, ...) HSE_PRINTF(2, 5);
 
+/* Convert a log priority level: numeric to string.
+ */
 const char *
 hse_logpri_val_to_name(hse_logpri_t val);
 
+/* Convert a log priority level: string to numeric.
+ */
 hse_logpri_t
 hse_logpri_name_to_val(const char *name);
 
+/* Support for custom "printf" format specifier such as "@@e" (for
```

Comment:
I think we should get rid of this personally. Just have APIs which take an merr_t.

Reply:
I think maybe we used to have log level names in our config and had to decode them???

Reply:
I agree.  The @@ custom specifiers are a bit ridiculous.